.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Unit Test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  typing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
This is unnecessary, as buiding does not depend on Git version history.

Reply:
I just based on the other workflow you added  in 33a81fa6cdc178a0511fff2a8cea56225ac5b343 and which had it . Should we remove it everywhere then?

Reply:
I was wrong; this uses `versioningit`, so we do need `fetch-depth: 0`.